www/caddy/src/etc/inc/plugins.inc.d/caddy.inc

```diff
@@ -26,41 +26,48 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-function caddy_services()
+function caddy_services(): array
```

Comment:
well, yes but it's not strict and if it ever becomes useful to return mixed types you need to change it again?